src/Vite.AspNetCore/Services/NodeScriptRunner.cs

```diff
@@ -21,7 +21,7 @@ internal sealed class NodeScriptRunner : IDisposable
 		/// <summary>
 		/// Initializes a new instance of the <see cref="NodeScriptRunner"/> class.
 		/// </summary>
-		public NodeScriptRunner(ILogger logger, string pkgManagerCommand, string scriptName, string workingDirectory, CancellationToken cancellationToken = default)
+		public NodeScriptRunner(ILogger logger, string pkgManagerCommand, string scriptName, string workingDirectory, Action<string>? onStandardOutputRead = null, CancellationToken cancellationToken = default)
```

Comment:
Consider making this an overload if binary compatibility with the previous version is a concern.

Reply:
It's not, this is all internal.